src/NServiceBus.Testing.Tests/Issue508.cs

```diff
@@ -1,12 +1,12 @@
 ﻿namespace NServiceBus.Testing.Tests
 {
     using System;
+    using System.Threading.Tasks;
     using MyMessages;
     using NUnit.Framework;
-    using Saga;
 
     [TestFixture]
-    public class Issue508 : BaseTests
+    public class Issue508
```

Comment:
I think we should get rid of these IssueXYZ tests. They are not obvious and if these tests add value, they should be with the rest of the tests.